main.py

```diff
@@ -0,0 +1,64 @@
+import logging
+
+from loan_prediction.config_loader import ConfigLoader
+from loan_prediction.data_loader import DataLoader
+from loan_prediction.data_processor import DataProcessor
+from loan_prediction.loan_classifier_model import LoanClassifierModel
+from loan_prediction.utils import plot_feature_importance, visualize_results
+
+logging.basicConfig(level=logging.DEBUG, format="%(asctime)s - %(levelname)s - %(message)s")
+logger = logging.getLogger(__name__)
+
+
+# Load configuration
+configurator = ConfigLoader()
+configurator.load()
+logger.debug(configurator.config_str())
+logger.info("Configuration loaded")
+
+
+# Load data
+filepath = "./data/sample/sample.csv"
```

Comment:
this will run locally.
have you also tried to load data from Databricks Volumes?